### ISSUE TITLE
Pyload fails to install on specific Ubuntu 20.04 image.

### ISSUE THREAD

**Issue by DonneyF** [swizzin/swizzin#677]
## Bug description
Pyload fails to install on Ubuntu 20.04, suspected due to specific build.

## Reproduction steps
1. Launch new Ampere ARM instance on Oracle Cloud
2. `sudo apt update && sudo apt upgrade`
3. Download and run the install script from README
4. `box install pyload`

## Installed applications through swizzin
pyload

## Expected behavior
Installation completes successfully

## Server Info
**(please complete the following information):**
 - OS: Ubuntu 20.04
 - Arch: arm64
 - Image Source: Oracle Cloud, build 2021.05.13-0

## Is Swizzin up to date?
Uses latest install script.

## Additional context
The install step "Initializing database" fails. The logs reveal:

 ```
 running build_ext
    building '_cffi_backend' extension
    creating build/temp.linux-aarch64-2.7
    creating build/temp.linux-aarch64-2.7/c
    aarch64-linux-gnu-gcc -pthread -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-Mlh1FU/python2.7-2.7.18=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -DUSE__THREAD -DHAVE_SYNC_SYNCHRONIZE -I/usr/include/ffi -I/usr/include/libffi -I/usr/include/python2.7 -c c/_cffi_backend.c -o build/temp.linux-aarch64-2.7/c/_cffi_backend.o
    c/_cffi_backend.c:15:10: fatal error: ffi.h: No such file or directory
       15 | #include <ffi.h>
          |          ^~~~~~~
    compilation terminated.
    error: command 'aarch64-linux-gnu-gcc' failed with exit status 1
    ----------------------------------------
    ERROR: Failed building wheel for cffi
 ```

## Solution

The Ubuntu image was missing the dependency `libffi-dev`. Installing this via aptitude and reinstalling pyload fixes this issue.

I'm not sure if we'd want to update the dependencies in the script, but I created this issue in case others encounter this in the future.

**Comment by flying-sausages** [comment]
Thanks! Do you know if this happens on normal Ubuntu for Arm as well? That one is supported, but I haven't seen this issue when i was testing it a couple months back.

**Comment by flying-sausages** [comment]
Oh never mind it's affecting Ubuntu for Arm as well. I'll take a look tomorrow.

**Comment by liaralabs** [comment]
Merged in #699
